### ISSUE TITLE
Material stepper validation

### ISSUE THREAD

**Issue by arfanotti** [ngx-formly/ngx-formly#2596]
**Why all formGuoup is validated upon editing a single Field?**

I'm having trouble with the validation of a custom template of a material vertical stepper: upon inserting a value, all form field are validated and the whole form and all the fields (even the one that are not required and, hence, valid) are marked as "invalid" until all invalid fields are corrected.

The desired functionality would be for the validation to be on the single field.

Here a  [stackblitz](https://stackblitz.com/edit/angular-apzubb) with a quick example. To produce the unwanted behaviour just write int the first field (First name) and then go to the second, because the second "step" contains a required field that is not filled, all the form is marked invalid.

Thanks in advance for your help.

**Comment by arfanotti** [comment]
Is this project dead?

**Comment by aitboudad** [comment]
it's not dead we just don't have enough time to provide the support for each request, our demo example is working as expected https://formly.dev/examples/advanced/multi-step-form so not sure why you've got that behavior, I may try to debug on my side at the end of the week if time allows

**Comment by aitboudad** [comment]
fix by #2644, the solution is to update the stepper declaration:

```patch

@NgModule({
  imports: [
    ...,
    FormlyModule.forRoot({
      types: [
-       { name: 'stepper', component: FormlyFieldStepper, wrappers: ['form-field'] },
+       { name: 'stepper', component: FormlyFieldStepper, wrappers: [] },
      ],
    }),
  ],
})
export class AppModule {}
```

Sorry about the delay and happy coding!
